build.mill

```diff
@@ -60,6 +62,27 @@ trait ScalaNativeCryptoModule extends Shared with Publish {
   def scalacOptions = super.scalacOptions() ++ Seq(
     "-P:scalanative:genStaticForwardersForNonTopLevelObjects"
   )
+
+  val osName = {
+    val _osName = Properties.osName.toLowerCase()
+    if (_osName.contains("mac")) "darwin"
+    else if (_osName.contains("windows")) "windows"
+    else "linux"
+  }
+  val archName = Properties.propOrEmpty("os.arch").toLowerCase() match {
+    case "amd64" => "x86_64" // try to follow llvm triple
+    case "arm64" => "aarch64"
+    case s       => s
+  }
```

Comment:
I would avoid this indirection to make java os.arch and osName to look like clang target triple values.

Reply:
Or follow the GCC triple? I copy similar condition codes from mill project to detect platform (win/linux/mac). But the triple is added by myself.